components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -43,6 +43,7 @@ type Props = {
     };
     autoTour: boolean;
     punchOut?: TutorialTipPunchout | null;
+    pulsatingDotPosition?: Coords | undefined;
```

Comment:
oh, during an initial implementation I tried to place the pulsatingDog to whatever place I wanted, even setting it from the calling/triggering component (to avoid having to use a redux call to communicate the location, etc, etc). I noticed that for this particular case it could potentially not work or overcomplicate things, however, I thought that maybe keeping that option to the pulsating dot to receive the position as coordinates could bring us some kind of future benefit. What do you think about this, since you are one of the people that has worked the most recently with these modules?

Reply:
I like the functionality of it, but I'd rather remove it since it doesn't have any current uses.

Reply:
I like it too :( , but you are right, the right way is to remove it 👍